packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.js

```diff
@@ -362,6 +363,13 @@ function InspectedElementView({
         });
       }
     };
+  } else if (type === ElementTypeContext) {
```

Comment:
Is this type just Provider or Consumer too?

Reply:
Pretty sure this was vor Provider and Consumer. It's working under the assumption the Context.Consumer won't have editable props to begin with. But even if it had any I don't see an issue making them editable?